installation/Makefile

```diff
@@ -0,0 +1,117 @@
+API_DIR = ../estela-api
+QUEUING_DIR = ../queueing
+WEB_DIR = ../estela-web
+APIPOD = $$(kubectl get pod -l app=estela-django-api -o jsonpath="{.items[0].metadata.name}")
+APIIP = $$(kubectl get services -n $${NAMESPACE} estela-django-api-service --output jsonpath='{.status.loadBalancer.ingress[0].ip}')
+
+
+ifeq ($(OS),Windows_NT)
+	DOCKER_COMPOSE = docker compose
+else
+	UNAME_S := $(shell uname -s)
+	ifeq ($(UNAME_S),Linux)
+		DOCKER_COMPOSE = docker-compose
+	else # OSX
+		DOCKER_COMPOSE = docker compose
+	endif
+endif
+
+
+.PHONY: resources
+resources:
+	-minikube start
+	-echo "$$(minikube ssh 'grep host.minikube.internal /etc/hosts | cut -f1')" > tmp
+	-python start-environment.py
+	-$(DOCKER_COMPOSE) up -d
+	-minikube delete
+	-. .env && minikube start \
```

Comment:
I think this has to be `. ./.env`. It wouldn't work on my machine (Linux) otherwise.